routes/orders.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const ordersRouter = express.Router();
+const {} = require('../db');
+const { getOrderById, getAllOrders, createOrder } = require('../db/orders');
+const { requireUser } = require('./utils');
+
+ordersRouter.get('/', async (req, res, next) => {
+    try {
+        const { isAdmin } = req.body;
```

Comment:
This is a bit insecure.  We should use isAdmin from the user data from the token that is passed.